CHANGELOG.md

```diff
@@ -23,6 +23,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Security
 - none yet
 
+## [3.0.0-beta29] - 2017-XX-XX
```

Comment:
@lespacedunmatin This date will be specified in a dedicated release commit, see https://github.com/cozy/cozy-ui/commit/bcf266c60b8e3278368b508cc8b389f2bd2bee3e for example. :)